web/src/components/contactForm.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import tw from 'twin.macro';
+
+const ContactWrapper = styled.div`
+    ${tw`text-center`}
+`;
+
+const StyledForm = styled.form`
+    ${tw`flex flex-col items-center`}
+`;
+
+const StyledHeading = styled.h1`
+    ${tw`font-bold mb-8 relative text-4xl text-black-3 leading-tight`}
+`;
+const InputWrapper = styled.div`
+    ${tw`flex gap-1`}
+`;
+
+const StyledButton = styled.button`
+    ${tw`cursor-pointer transition duration-500 ease-out bg-green py-2 px-6 text-white max-w-max mt-16 text-xl tracking-widest hover:bg-hoverGreen`}
+`;
+
+const StyledInput = styled.input`
+    ${tw`border-2`}
+`;
+
+const StyledLabel = styled.label`
+    ${tw`flex flex-col mb-4 mx-4 text-sm`}
+`;
```

Comment:
The form input styling feels a little small. Can you match the styled from the oldIndex page? I'd also remove the colons from the labels and left align the text (probably need `align-items: flex-start` on the wrapper.